### PR TITLE
[master] Switch all VERSIONS to be STATIC_VERSION

### PR DESCRIPTION
**Pull request by seemethere**
`STATIC_VERSION` is used to denote development builds as well.

### CODE REVIEW

**Comment by tiborvass** [comment]
I'm not sure I understand all of it, but it looks to be consistent with the rest so lgtm. But probably needs another review.

File: static/Makefile [hunk 1]
```diff
@@ -53,7 +53,7 @@ cross-mac: cross-all-cli ## create tgz with darwin x86_64 client only
 cross-win: cross-all-cli cross-win-engine ## create zip file with windows x86_64 client and server
 	mkdir -p build/win/docker
 	cp $(CLI_DIR)/build/docker-windows-amd64 build/win/docker/docker.exe
-	cp $(ENGINE_DIR)/bundles/cross/windows/amd64/dockerd-$(VERSION).exe build/win/docker/dockerd.exe
+	cp $(ENGINE_DIR)/bundles/cross/windows/amd64/dockerd-$(STATIC_VERSION).exe build/win/docker/dockerd.exe
```
Comment (thaJeztah) [inline static/Makefile]: Do we actually use `VERSION` now somewhere? If not, could we set the value to `$VERSION` directly? (`STATIC_VERSION` is a bit confusing)

Comment (seemethere) [reply static/Makefile]: We have the same standard for Deb and rpm versioning as well since they do get transformed from the regular version.

Comment (thaJeztah) [reply static/Makefile]: ah, I see; those use `DEB_VERSION`. Gotcha

**Pull request merged**